src/main/java/frc/trigon/robot/commands/Commands.java

```diff
@@ -70,4 +84,125 @@ public static Command duplicate(Command command) {
+    }
+
+    public static Command getCloseSpeakerShotCommand() {
+        return new ParallelCommandGroup(
+                getPrepareCloseSpeakerShotCommand(),
+                getFeedNoteForCloseSpeakerShotCommand()
+        );
+    }
+
+    public static Command getPrepareCloseSpeakerShotCommand() {
+        return new ParallelCommandGroup(
+                PitcherCommands.getSetTargetPitchCommand(PitcherConstants.CLOSE_SHOT_PITCH),
+                ShooterCommands.getSetTargetVelocityCommand(ShooterConstants.CLOSE_SHOT_VELOCITY_ROTATIONS_PER_SECOND, ShooterConstants.CLOSE_SHOT_VELOCITY_ROTATIONS_PER_SECOND * ShooterConstants.RIGHT_MOTOR_TO_LEFT_MOTOR_RATIO)
+        );
+    }
+
+    public static Command getFeedNoteForCloseSpeakerShotCommand() {
+        return runWhen(IntakeCommands.getSetTargetStateCommand(IntakeConstants.IntakeState.FEED_SHOOTING), () -> RobotContainer.SHOOTER.atTargetVelocity() && RobotContainer.PITCHER.atTargetPitch());
+    }
+
+    public static Command getAutonomousScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getAutonomousPrepareForAmp(),
+                runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                getPathfindToAmpCommand().andThen(duplicate(CommandConstants.FACE_AMP_COMMAND))
+        );
+    }
+
+    public static Command getScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getPrepareForAmpCommand(),
+                runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                duplicate(CommandConstants.FACE_AMP_COMMAND)
+        );
+    }
+
+    public static Command getAutonomousPrepareForAmp() {
+        return new ParallelCommandGroup(
+                runWhen(
+                        AmpAlignerCommands.getSetTargetStateCommand(AmpAlignerConstants.AmpAlignerState.OPEN)
+                                .alongWith(PitcherCommands.getSetTargetPitchCommand(PitcherConstants.AMP_PITCH)),
+                        () -> RobotContainer.POSE_ESTIMATOR.getCurrentPose().getTranslation().getDistance(
+                                FieldConstants.IN_FRONT_OF_AMP_POSE.get().getTranslation()) < FieldConstants.MINIMUM_DISTANCE_FROM_AMP_FOR_AMP_PREPARATION_METERS
+                ),
+                ShooterCommands.getSetTargetVelocityCommand(ShooterConstants.AMP_SHOOTING_VELOCITY_ROTATIONS_PER_SECOND, ShooterConstants.AMP_SHOOTING_VELOCITY_ROTATIONS_PER_SECOND)
+        );
+    }
+
+    public static Command getPrepareForAmpCommand() {
+        return new ParallelCommandGroup(
+                AmpAlignerCommands.getSetTargetStateCommand(AmpAlignerConstants.AmpAlignerState.OPEN),
+                PitcherCommands.getSetTargetPitchCommand(PitcherConstants.AMP_PITCH),
+                ShooterCommands.getSetTargetVelocityCommand(ShooterConstants.AMP_SHOOTING_VELOCITY_ROTATIONS_PER_SECOND, ShooterConstants.AMP_SHOOTING_VELOCITY_ROTATIONS_PER_SECOND)
```

Comment:
Maybe have a `getSetTargetVelocityCommand(double targetVelocityBothMotors)` for this?